### PR TITLE
feature(pu): add Dockerfile and its usage instructions

### PR DESCRIPTION
**Pull request by puyuan1996**
- add LightZero Dockerfile and its usage instructions

### CODE REVIEW

**Comment by jayyoung0802** [comment]
Docker test is normal！

File: README.zh.md [hunk 1]
```diff
@@ -135,6 +135,32 @@ pip3 install -e .
...
+
+2. **准备构建上下文**：在您的本地机器上创建一个新的空目录，将 Dockerfile 移动到此目录，并导航到此目录。这一步有助于在构建过程中避免向 Docker 守护进程发送不必要的文件。
+    ```bash
+    mkdir lightzero-docker
+    mv Dockerfile lightzero-docker/
+    cd lightzero-docker/
+    ```
+3. **构建 Docker 镜像**：使用以下命令构建 Docker 镜像。此命令应在包含 Dockerfile 的目录内运行。
+    ```bash
+    docker build -t ubuntu-py38-lz .
```
Comment (jayyoung0802) [inline README.zh.md:153]: docker build -t ubuntu-py38-lz:v0 -f ./Dockerfile .

File: README.zh.md [hunk 2]
```diff
@@ -135,6 +135,32 @@ pip3 install -e .
...
+    ```
+4. **从镜像运行容器**：使用以下命令以交互模式启动一个 Bash shell 的容器。
+    ```bash
+    docker run -it --rm ubuntu-py38-lz /bin/bash
```
Comment (jayyoung0802) [inline README.zh.md:157]: docker run -dit ubuntu-py38-lz:v0 /bin/bash

File: README.zh.md (hunk 1 above)
Comment (puyuan1996) [reply README.zh.md:153]: polished now

**Pull request merged**